main.cpp

```diff
@@ -82,32 +82,54 @@ send me a DM to check your pull request
 
 /*
  */
+#include<iostream>
+
 struct MidiKeyboard
 {
     int keys = 88;
     bool pitchWheel = true;
     bool modWheel = true;
-    bool usbCompatable = true;
-    bool metalBody = false;
+    bool keyPressed = false;
     
     void playMidiNotes(bool keyPressed);
-    void changePitch(bool pitchWheelUp);
-    void controlMacros(bool modWheelUp);
+    void changePitch(bool pitchWheel);
+    void controlMacros(bool modWheel);
 };
 
 void MidiKeyboard::playMidiNotes(bool)
 {
-  
+    if(keyPressed == true)
+    {
+        std::cout<<"key pressed"<<std::endl;  
+    }
+    else
+    {
+        std::cout<<"key not pressed"<<std::endl;
```

Comment:
add spaces around your `<<` to make this more readable. 
- [ ] fix the formatting to improve readability.